frontend/workflows/projectCatalog/src/catalog/index.tsx

```diff
@@ -173,6 +168,7 @@ const Catalog: React.FC<WorkflowProps> = ({ heading }) => {
                 true
               );
             }}
+            size="medium"
```

Comment:
jst curious is this needed? didn't see a change locally without or without it

Reply:
Nope, because its defaulted in core, I think I just accidentally added it :)